src/View/Helper/FormHelper.php

```diff
@@ -41,15 +41,18 @@ class FormHelper extends Helper
      */
     protected $_templates = [
         'dateWidget' => '<ul class="list-inline"><li class="year">{{year}}</li><li class="month">{{month}}</li><li class="day">{{day}}</li><li class="hour">{{hour}}</li><li class="minute">{{minute}}</li><li class="second">{{second}}</li><li class="meridian">{{meridian}}</li></ul>',
-        'error' => '<div class="help-block">{{content}}</div>',
-        'help' => '<div class="help-block">{{content}}</div>',
+        'error' => '<div class="invalid-feedback">{{content}}</div>',
+        'label' => '<label{{attrs}}>{{text}}{{tooltip}}</label>',
+        'help' => '<small{{attrs}} class="form-text text-muted">{{content}}</small>',
+        'tooltip' => '<span data-toggle="tooltip" title="{{content}}" class="glyphicon glyphicon-info-sign"></span>',
         'inputContainer' => '<div class="form-group {{type}}{{required}}">{{content}}{{help}}</div>',
-        'inputContainerError' => '<div class="form-group {{type}}{{required}} has-error">{{content}}{{error}}{{help}}</div>',
-        'radioInlineFormGroup' => '{{label}}<div class="radio-inline-wrapper">{{input}}</div>',
-        'radioNestingLabel' => '<div class="radio">{{hidden}}<label{{attrs}}>{{input}}{{text}}</label></div>',
-        'staticControl' => '<p class="form-control-static">{{content}}</p>',
+        'inputContainerError' => '<div class="form-group {{type}}{{required}} is-invalid">{{content}}{{error}}{{help}}</div>',
+        'radioInlineFormGroup' => '{{label}}<div class="form-check form-check-inline">{{input}}</div>',
+        'radioNestingLabel' => '<div class="form-check">{{hidden}}<label{{attrs}}>{{input}}{{text}}{{tooltip}}</label></div>',
+        'staticControl' => '<input type="text" readonly class="form-control-plaintext" value="{{content}}>">',
```

Comment:
Static control needs to stay a text element as earlier. As mention in [BS4 migration](https://getbootstrap.com/docs/4.0/migration/#forms-1) `form-control-static` has been renamed to `form-control-plaintext`, so keep the existing template and just change class name.

Reply:
Why does it need to be a `<p>`-Tag? Official [Doc's](https://getbootstrap.com/docs/4.0/components/forms/#readonly-plain-text) use the `<input>`-Tag with `form-control-plaintext` class.

Reply:
Using input tag would allow the value to be manipulated through JavaScript. Currently the `staticControl()` generates a P tag and a hidden input with the value, so when using security component the value cannot be manipulated.